src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -183,7 +195,10 @@ export class SearchResults extends Component {
 
     return (
       <>
-        <h2 className="vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal">
+        <h2
+          data-forms-focus
+          className="vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal"
```

Comment:
This is my OCD kicking in but would you want to put `data-forms-focus` after `className` to be alphabetical, just like we did [here](https://github.com/department-of-veterans-affairs/vets-website/pull/12036/files#diff-af925777d60ec827d9db7a2d9c1ee02bR168)? 😅

Reply:
Haha, I like the OCD!

Reply:
Since I haven't had a chance in the past two weeks to get my SOCKS proxy set up, would you please have Trevor review? I checked with him and he can do it late this afternoon.